app/tests/api/routes/test_health.py

```diff
@@ -5,10 +5,80 @@ def test_bmi(client: TestClient):
     # TODO: Add more test cases
```

Comment:
`nit` can you remove this outdated comment?

Reply:
Yes!